admin/test/system/workarea/admin/publish_authorization_system_test.rb

```diff
@@ -51,7 +51,7 @@ def test_user_cannot_select_publish_now_in_workflows
         click_link 'HTML'
         click_button 'create_block'
 
-        assert(page.has_content?('Success'))
+        assert_text('Success')
```

Comment:
This is a bit of an unrelated change.

Reply:
this was for debugging, the test here failed and i'm still not 100% on why.